src/main/java/org/broadinstitute/hellbender/tools/exome/NormalizeSomaticReadCounts.java

```diff
@@ -191,22 +191,31 @@ private void applyTangentNormalization(final PoN pon, final ReadCountCollection
         // Output the tangent normalized counts.
         final ReadCountCollection tangentNormalized = targetMapper.fromPoNtoCaseCountCollection(tangentNormalizedCounts, originalCounts.columnNames());
         writeTangentNormalizedOutput(tangentNormalized);
-        writePreTangentNormalizationCounts(targetMapper, tangentNormalizationInputCounts, originalCounts.columnNames());
+        final ReadCountCollection preTangentNormalized = targetMapper.fromPoNtoCaseCountCollection(tangentNormalizationInputCounts, originalCounts.columnNames());
+        writePreTangentNormalizationOutput(preTangentNormalized);
         writeTangentBetaHats(tangentBetaHats, originalCounts.columnNames());
     }
 
     /**
-     * Writes the pre-tangent-normalization read counts if a file was provided for it.
+     * Writes the tangent-factor normalized outputs.
```

Comment:
Is this comment correct?  I think you accidentally missed a change from a cut and paste.